pkgs/top-level/qt5-packages.nix

```diff
@@ -271,6 +271,8 @@ in (noExtraAttrs (kdeFrameworks // plasmaMobileGear // plasma5 // plasma5.thirdP
 
   sierra-breeze-enhanced = callPackage ../data/themes/kwin-decorations/sierra-breeze-enhanced { useQt5 = true; };
 
+  slicer3d = callPackage ../applications/science/biology/slicer3d { };
```

Comment:
The Qt 5 package set is for *components* relating to the Qt framework and not downstream applications. This should be in toplevel and moved to pkgs/by-name.